backend/scripts/run_worker.sh

```
@@ -0,0 +1,20 @@
+PYTHON_SCRIPT="$SCRIPT_DIR/run_worker.py"
+
+# Check for worker type argument
+if [ -z "$1" ]; then
+    echo "Usage: ./run_worker.sh <ingestion|dump|notification>"
+    exit 1
+fi
+
+WORKER_TYPE=$1
+
+# Change to backend directory
+cd "$BACKEND_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)